apps/ui/src/pages/applications/sections/institutional.tsx

```diff
@@ -0,0 +1,265 @@
+							<InputBox
+								label={translate('institutional-section.form.researcherProfile')}
+								subLabel={translate('institutional-section.form.researcherProfileLabel')}
+								name="institutionalProfileUrl"
+								placeHolder="https://"
+								control={control}
+								rule={rule}
+								required
+								disabled={!isEditMode}
+							/>
+						</Col>
+					</Row>
+					<Row gutter={26}>
+						<Col xs={{ flex: '100%' }} md={{ flex: '100%' }} lg={{ flex: '50%' }}>
+							<InputBox
+								label={translate('institutional-section.form.positionTitle')}
+								name="institutionalPositionTitle"
+								control={control}
+								rule={rule}
+								required
+								disabled={!isEditMode}
+							/>
+						</Col>
+					</Row>
+				</SectionContent>
+				<SectionContent title={translate('institutional-section.section2')}>
+					<Row gutter={26}>
+						<Col xs={{ flex: '100%' }} md={{ flex: '100%' }} lg={{ flex: '50%' }}>
+							<InputBox
+								label={translate('institutional-section.form.country')}
+								name="institutionCountry"
+								control={control}
+								rule={rule}
+								required
+								disabled={!isEditMode}
+							/>
+						</Col>
+					</Row>
+					<Row gutter={26}>
+						<Col xs={{ flex: '100%' }} md={{ flex: '100%' }} lg={{ flex: '50%' }}>
+							<InputBox
+								label={translate('institutional-section.form.streetAddress')}
+								name="institutionStreetAddress"
+								control={control}
+								rule={rule}
+								required
+								disabled={!isEditMode}
+							/>
+						</Col>
+						<Col xs={{ flex: '100%' }} md={{ flex: '100%' }} lg={{ flex: '50%' }}>
+							<InputBox
+								label={translate('institutional-section.form.building')}
+								name="institutionBuilding"
+								control={control}
+								rule={rule}
+								disabled={!isEditMode}
+							/>
+						</Col>
+					</Row>
+					<Row gutter={26}>
+						<Col xs={{ flex: '100%' }} md={{ flex: '100%' }} lg={{ flex: '50%' }}>
+							<InputBox
+								label={translate('institutional-section.form.state')}
+								name="institutionState"
+								control={control}
+								rule={rule}
+								required
+								disabled={!isEditMode}
+							/>
+						</Col>
+						<Col xs={{ flex: '100%' }} md={{ flex: '100%' }} lg={{ flex: '50%' }}>
+							<InputBox
+								label={translate('institutional-section.form.city')}
+								name="institutionCity"
+								control={control}
+								rule={rule}
+								required
+								disabled={!isEditMode}
+							/>
+						</Col>
+					</Row>
+					<Row gutter={26}>
+						<Col xs={{ flex: '100%' }} md={{ flex: '100%' }} lg={{ flex: '50%' }}>
+							<InputBox
+								label={translate('institutional-section.form.postalCode')}
+								name="institutionPostalCode"
+								control={control}
+								rule={rule}
+								required
+								disabled={!isEditMode}
+							/>
+						</Col>
+					</Row>
+				</SectionContent>
+				<SectionFooter />
+			</Form>
+		</SectionWrapper>
```

Comment:
I also noticed the designs show a 'Back' button so if it's easy this may be a good time to add one